Population/Population Datamart/Create R Population Datamart Templates.R

```diff
@@ -0,0 +1,58 @@
+### 1 - Information ----
```

Comment:
Can we rename the file name with "_ONEOFF" as it doesn't need to be run everytime?